components/AdvancedOptionsItem.qml

```diff
@@ -22,6 +22,7 @@ RowLayout {
         MoneroComponents.Label {
             id: title
             fontSize: 14
+            tooltipIconVisible: true
```

Comment:
What is this used for? I wasn't able to figure it out.

Reply:
When tooltipIconVisible = true we display the question mark icon that when hovered turns tooltip visible:
![image](https://user-images.githubusercontent.com/45968869/119907044-bd0d0200-bf4f-11eb-8a41-83368257f751.png)

When tooltipIconVisible = false (default), we display only the tooltip box.

Reply:
Ah I looked on the wrong page :)